task09-react/src/components/dataSectionComponents/cardsComponents/_islandCard.scss

```diff
@@ -0,0 +1,126 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary styles for the island card components
+ */
+.island {
+  &__card {
+    position: relative;
+    width: 300px;
+    height: 300px;
+    min-width: 300px;
+    min-height: 300px;
+    &__textContainer {
+      position: absolute;
+
+      border-radius: 6px;
+      background-color: rgba($color-dark-02, 0.59);
+      padding: 0px 24px !important;
+      margin: 12px !important;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      backdrop-filter: blur(6px);
+    }
+
+    &__locationContainer {
+      display: flex;
+
+      #placeIcon {
+        fill: white;
+        width: 16px;
+      }
+    }
+
+    &__location {
+      color: #f4f4f4;
+      font-family: "Roboto";
+      font-style: normal;
+      font-weight: 400;
+      font-size: 18px;
+      line-height: 24px;
+      letter-spacing: 0.15px;
+    }
+
+    &__name {
+      color: white;
+      letter-spacing: 0.1px;
+      font-family: "Roboto Slab";
+      font-style: normal;
+      font-weight: 700;
+      font-size: 18px;
+      line-height: 24px;
+    }
+
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+      border-radius: inherit;
+    }
+  }
+}
+
+@include media-breakpoint-down(lg) {
```

Comment:
use a mobile first approach, set the mobile styles first and override them with the styles for larger screens